bitnami/airflow/templates/deployment-scheduler.yaml

```diff
@@ -201,6 +205,10 @@ spec:
       {{- if .Values.airflow.cloneDagFilesFromGit.enabled }}
         - name: git-cloned-dag-files
           emptyDir: {}
+        - name: dag-git-keys
+          secret:
+            secretName: {{ .Values.airflow.cloneDagFilesFromGit.sshKeySecret }}
+            defaultMode: 0600
```

Comment:
This volume should only exist if `airflow.cloneDagFilesFromGit.sshKeySecret` is set.